pallets/pips/src/benchmarking.rs

```diff
@@ -0,0 +1,377 @@
+// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
+// Copyright (c) 2020 Polymath
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(feature = "runtime-benchmarks")]
+use crate::*;
+use frame_benchmarking::benchmarks;
+use frame_support::traits::UnfilteredDispatchable;
+use frame_system::RawOrigin;
+use pallet_identity::{self as identity, benchmarking::make_account};
+use polymesh_common_utilities::{MaybeBlock, SystematicIssuers};
+use sp_std::{
+    convert::{TryFrom, TryInto},
+    prelude::*,
+};
+
+const DESCRIPTION_LEN: usize = 1000;
+const URL_LEN: usize = 500;
+
+pub fn make_proposal<T: Trait>(content_len: usize) -> (Box<T::Proposal>, Url, PipDescription) {
+    let content = vec![b'X'; content_len];
+    let proposal = Box::new(frame_system::Call::<T>::remark(content).into());
+    let url = Url::try_from(vec![b'X'; URL_LEN].as_slice()).unwrap();
+    let description = PipDescription::try_from(vec![b'X'; DESCRIPTION_LEN].as_slice()).unwrap();
+    (proposal, url, description)
+}
+
+benchmarks! {
+    _ {}
+
+    set_prune_historical_pips {
+        let origin = RawOrigin::Root;
+    }: _(origin, true)
+    verify {
+        assert_eq!(true, PruneHistoricalPips::get());
+    }
+
+    set_min_proposal_deposit {
+        let origin = RawOrigin::Root;
+        let deposit = 0.into();
```

Comment:
Let's use something other than 0 since 0 is the default value. Not sure if the substrate/rockdb's middleware is smart enough yet but this is a no-op and does not require writing to the db.

Same concern with other default/0 values you have used in the benchmarks.